### PR TITLE
Remove Redis and Jdbc modules

### PR DESCRIPTION
**Pull request by otrosien**
They have been deprecated since https://github.com/zalando-nakadi/fahrschein/pull/294 and now it's time for removal.

> Nakadi's low-level API has one use-case: if you always want to read a
> topic from the beginning. This is incompatible with the idea of an
> external persistant storage for offsets. If offset persistance is
> needed, people should use the normal subscription APIs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,20 +86,11 @@ nakadiClient.stream(eventName).skipUnavailableOffsets(partitions);
 *Please do not use the Low-level API, as it is deprecated by Nakadi.*
 
 The Low-level API requires local persistence of partition offsets.
-There are currently three persistent `CursorManager` implementations: InMemory, Postgres and Redis.
-
-!!! warning
-    Postgres and Redis cursor managers are DEPRECATED and will be
-    removed in an upcoming version of Fahrschein.
+There is one `CursorManager` implementation left: InMemory. Postgres and Redis
```
Comment (othon2000) [inline README.md:89]: Duplicate `Postgres and Redis` (see below line)

File: README.md [hunk 2]
```diff
@@ -110,16 +101,11 @@ nakadiClient.stream(eventName)
...
-
-Fahrschein-JDBC provides Flyway schema migrations in the `fahrschein-db` resource folder. You can
-point Flyway to it using the [`resources`](https://flywaydb.org/documentation/configuration/parameters/locations) configuration parameter.
-
 ## Using multiple partitions and multiple consumers
 
 With the `PartitionManager` api it is possible to coordinate between multiple nodes of one application, so that only one node is consuming events from a partition at the same time.
 
-Partitions are locked by one node for a certain time. This requires that every node has an unique name or other identifier.
+Partitions are locked by one node for a certain time. This requires that every node has a unique name or other identifier.
```
Comment (othon2000) [inline README.md:108]: As you already made a grammar fix, here's another one:
`This requires that every node has` -> `This requires that every node have`

**Review by othon2000 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (otrosien) [inline README.md:89]:
```suggestion
There is one `CursorManager` implementation left: InMemory.
```

File: README.md (hunk 2 above)
Comment (otrosien) [reply README.md:108]: sure about that one?

Comment (otrosien) [reply README.md:108]: maybe "This requires every node to have a unique name or other identifier."?

Comment (othon2000) [reply README.md:108]: I'm sure about my suggestion, but what you wrote also works.

**Review by othon2000 — APPROVED** [review]

**Comment by othon2000** [comment]
:+1:

**Comment by otrosien** [comment]
👍

**Comment by otrosien** [comment]
👍

**Comment by MALPI** [comment]
👍

**Comment by MALPI** [comment]
👍

**Pull request merged**